packages/main/test/specs/TextArea.spec.js

```diff
@@ -56,6 +56,22 @@ describe("Attributes propagation", () => {
 		assert.notOk(await innertextAreaWarning.getAttribute("aria-invalid"), "aria-invalid is not rendered");
 		assert.strictEqual(await innertextAreaError.getAttribute("aria-invalid"), "true", "aria-invalid is set to true");
 	});
+
+	it("Tests aria-label is set to match the label text when label is for that text area", async () => {
+		const textArea = await browser.$("#taWithLabelID").shadow$("textarea");
+
+		const EXPECTED_ARIA_LABEL = "This should be in aria-label property of the text area";
```

Comment:
It would be better to get the text of the label instead of comparing it to a hard coded text.

Reply:
done